scripts/functions-deploy-tests/tests.ts

```diff
@@ -260,6 +263,83 @@ describe("firebase deploy", function (this) {
     }
   });
 
+  it("skips duplicate deploys functions with runtime options", async () => {
+    const opts: Opts = {
```

Comment:
It looks like this configuration must match the configuration set in the previous test case in order for the test case to work as intended.

Can we make that coupling explicit? e.g. pull out configuration as a test variable.

Reply:
They don't need to match the previous one, I was just copy/pasting. I updated the opts to not be the giant list (to reduce distractions from the purpose of the test.)